grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -167,14 +167,25 @@ def get_aoi(vector=None):
     if vector:
         args['input'] = vector
 
+    topology = gs.parse_command('v.info', map=vector, flags='t')
```

Comment:
There is `vector_info_topo()` in GRASS scripting library. Then no casting is needed.

Reply:
ok, will check!